src/include/catalog/postgres/nameBuilder.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <string>
+
+#include "catalog/postgres/pg_defs.h"
+
+namespace terrier::catalog::postgres {
```

Comment:
I don't think that this needs to be under the Postgres name space

Reply:
I discussed with Matt on Friday about this, and decided to put it there since this utility is similar to what Postgres has